eng/BannedSymbols.txt

```diff
@@ -0,0 +1,3 @@
+M:Moq.Mock`1.#ctor; Pass MockBehavior.Strict explicitly to the constructor for Mock<T>
```

Comment:
Hear, hear! I've been encouraging this manually in PRs, but am always in favor of automating policy.